doc/changes/changes_2.1.0.md

```diff
@@ -42,5 +52,18 @@ Version: 2.1.0
 * #295: Made notebook-tests mandatory for merge
 * #193: Ignored warnings in notebook tests
 * #297: Reduced log level for transitive libraries in notebook tests
-* #307: Made the notebook tests running in parallel;
-        moved common steps from test jobs to a composite action
+* #307: Made the notebook tests running in parallel; moved common steps from test jobs to a composite action
+* #308: Removed redundant dependencies from file `notebook_requirements.txt`.
+
+## Dependency Updates
+
+### Jupyter Environment Dependencies
+
+In File [notebook_requirements.txt](https://github.com/exasol/ai-lab/blob/main/exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook_requirements.txt):
+* Removed dependency to `uncertainties`
+* Updated `scikit-learn:1.0.2` to `1.5.1`
+* Updated `matplotlib:3.7.4` to `3.9.2`
+* Updated `jupysql:0.10.10` to `0.10.12`
+* Relaxed `stopwatch.py:2.0.1` dependency to `2.*` to avoid inconsistencies with ITDE
+* Replaced GitHub dependency to `exasol/notebook-connector:main` by pypi dependency `0.2.9`
```

Comment:
Are you sure this was at the last release also a GitHub dependencies to main

Reply:
The latest AI-Lab 2.0.0 release pointed to pypi release 0.2.8 of the notebook connector in file [notebook_requirements.txt](https://github.com/exasol/ai-lab/blob/2.0.0/exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook_requirements.txt). 

From that I guess, you propose to not mention this in the changes file, right?